src/inventory.cpp

```diff
@@ -271,6 +286,17 @@ std::string ItemStack::getShortDescription(const IItemDefManager *itemdef) const
 	return desc;
 }
 
+f32 ItemStack::getRange(const IItemDefManager *itemdef) const
+{
+    std::string str_meta_range = metadata.getString("range");
+    f32 range = getDefinition(itemdef).range;
+    bool is_valid;
+    float meta_range = string_to_float(str_meta_range, is_valid);
+    if (is_valid && meta_range > -1)
```

Comment:
Why is there no warning about the invalid range value?

Reply:
Are you asking to have a comment added for a warning?

Reply:
I'm asking why nothing is printed to the log. Invalid meta seems like something that should generate at least a warning.